exercise-3/src/com/nayema/Main.java

```diff
@@ -0,0 +1,16 @@
+package com.nayema;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please enter a number.");
+        int number = scanner.nextInt();
+
+        for (int i = 1; i <= number; i ++) {
```

Comment:
As mentioned, we usually use `int i = 0` when starting the iteration of the loop and `i < upperBound` for the condition instead of `1` and `<=`.